semgrep/tests/e2e/snapshots/test_json_output/test_default_rule__file/results.json

```diff
@@ -0,0 +1 @@
+{"results": [{"check_id": "eqeq-is-bad", "path": "targets/basic/stupid.py", "start": {"line": 3, "col": 12}, "end": {"line": 3, "col": 26}, "extra": {"message": "useless comparison operation `a+b == a+b` or `a+b != a+b`; possible bug?", "metavars": {"$X": {"start": {"line": 3, "col": 12, "offset": 55}, "end": {"line": 3, "col": 17, "offset": 60}, "abstract_content": "a+b", "unique_id": {"type": "AST", "md5sum": "07d71d85769e594dba9d7ae3d295c01f"}}}, "metadata": {}, "severity": "ERROR"}}, {"check_id": "javascript-basic-eqeq-bad", "path": "targets/basic/stupid.js", "start": {"line": 3, "col": 13}, "end": {"line": 3, "col": 19}, "extra": {"message": "useless comparison", "metavars": {"$X": {"start": {"line": 3, "col": 13, "offset": 24}, "end": {"line": 3, "col": 14, "offset": 25}, "abstract_content": "x", "unique_id": {"type": "id", "value": "x", "kind": "Global", "sid": 1}}}, "metadata": {}, "severity": "ERROR"}}], "errors": []}
```

Comment:
Note to self, find out why this printed message changed compared to the previous committed JSON.

Reply:
Ah of course! I changed this test to just test whether the default semgrep rules location is picked up correctly, and split the `--generate-config` test to a separate file. So this now uses eqeq.yml instead of the semgrep-rules template.